src/components/layouts/index.js

```diff
@@ -36,7 +38,9 @@ const useHandleAuthentication = () => {
 
   useEffect(() => {
     if (!getUserQuery.data) return;
+    const { id, email, username } = getUserQuery.data;
     setCookie('user', getUserQuery.data);
+    setSentryUser({ id, email, username });
```

Comment:
@bertramakers  @simon-debruijn  are we allowed to store email addresses in Sentry?
I mean is this GDPR compliant?

Reply:
Good question, I'm googleing about it. But @bertramakers do you use this feature too on e.g. udb3-silex?

Reply:
When following GDPR, aren't we also obligated to add a cookie-banner to be able to save user data to the cookies?

Reply:
When you use cookies, you need a cookie banner. So yes.

Reply:
A cookie banner is something to add to the backlog then. But setting the user here for Sentry, idk, I haven't found a clear answer on if we can just do this.

Reply:
These tags are added for UDB3 backend, either with custom tags are default Sentry tags:
<img width="325" alt="Screenshot 2021-03-26 at 09 27 47" src="https://user-images.githubusercontent.com/1322949/112603694-a47c5080-8e15-11eb-8bdd-0792cdee2f95.png">

Reply:
Maybe we just save the id? That doesn't reveal to much information

Reply:
I think we can avoid the cookie banner inside the app because we have a signup process and during the signup process you give a consent for cookies. But I'm not an expert in this field, so better check with someone from "legal" 😅

Reply:
Jup, just the `id` would be better I guess

Reply:
See [this commit](https://github.com/cultuurnet/udb3-frontend/pull/211/commits/982ce8c8fd947cd8bdb3c5dcb73a4d69b06818d1)

Reply:
Wow there's a lot going on in this thread 😅 

> no email address is stored in Sentry

@LucWollants we do currently store the email address from the backend Sentry calls, see for example https://sentry.io/organizations/publiq-vzw/issues/2287500812/?project=5435528&query=is%3Aunresolved (it's not a tag, it's user info)

> are we allowed to store email addresses in Sentry?
> I mean is this GDPR compliant?

@brampauwelyn Technically I think yes it's compliant. In the sense that we already have that info when the user signs up and we are allowed to process it. Sentry itself is also GDPR compliant. However, I agree that storing only the id might be better because while Sentry is a closed system and only accessible with a login, it is possible to make issues public to share them and then we might accidentally expose personally identifiable information. And having the email in Sentry is probably not that necessary now that I think about it. In the case that we really need to identify a particular user we can look up their id in Auth0.

> When following GDPR, aren't we also obligated to add a cookie-banner to be able to save user data to the cookies?

@simon-debruijn 
"When following GDPR" -> We always have to follow GDPR.
Storing the email address has nothing to do with cookies though.

But about a cookie banner in general: I agree with @LucWollants that a cookie banner is probably not necessary since you need to login first. However, I don't see a cookie consent on Auth0/CP right now. Shouldn't we add one there instead? (@brampauwelyn ?)

The only other cookie I know of that we store before logging in is the selected language. So maybe we should just find a solution for that. A cookie banner won't necessarily solve that because right now we set that language cookie immediately once you visit the landing page, while even with a cookie banner we shouldn't do that but wait for the consent instead.

Btw maybe we should discuss privacy- and security-sensitive questions like this on Slack in the future 😉